forge-gui/res/cardsfolder/upcoming/raddic_tal_zealot.txt

```diff
@@ -0,0 +1,11 @@
+Name:Raddic, Tal Zealot
+ManaCost:2 W B
+Types:Legendary Creature Human Knight
+PT:4/4
+K:Hexproof:Card.Black:white
+K:Hexproof:Card.Black:black
+T:Mode$ AttackersDeclared | ValidAttackers$ Knight.YouCtrl |Execute$ TrigDraft | TriggerZones$ Battlefield | TriggerDescription$ Whenever one or more Knights you control attack, draft a card from CARDNAME's spellbook.
+SVar:TrigDraft:DB$ Draft | Spellbook$ Midnight Reaper,Guardian of Faith,Knight of the Ebon Legion,Cavalier of Dawn,Benalish Marshal,Cavalier of Night,Murderous Rider,Order of Midnight,Acclaimed Contender,Dauntless Bodyguard,Valiant Knight,Smitten Swordmaster,Blacklance Paragon,History of Benalia,The Circle of Loyalty
+DeckHints:Type$Knight
+DeckHas:Type$Saga|Enchantment|Vampire|Artifact|Zombie|Elemental & Ability$Graveyard|LifeGain|Token|Counters
+Oracle:Hexproof from white, Hexproof from black.\nWhenever one or more Knights you control attack, draft a card from Raddic, Tal Zealot's spellbook.
```

Comment:
capitalization for the hexproof looks wrong

Reply:
Oh, gross... Arena's way of doing this is different than paper *sigh* this is probably good enough

just get rid of the period after `black.`